src/pages/resultsView/survival/SurvivalChart.tsx

```diff
@@ -410,10 +414,50 @@ export default class SurvivalChart extends React.Component<ISurvivalChartProps,
         return this.victoryLegendData.length > 0;
     }
 
+    @observable.shallow zoomDomain: any = {x: [0, this.initialRangeMaxValue]};
+
+    @autobind
+    private onZoom(domain: any, props: any) {
+        this.zoomDomain = {x: domain.x};
+    }
+
+    @autobind
+    private updateDomain(domain: any) {
+        this.zoomDomain = {x: domain.x};
+    }
+
+    @computed get maximumDataMonthValue() {
+        let max = Number.MIN_VALUE;
+        _.forEach(this.sortedGroupedSurvivals, (survivals) => {
```

Comment:
_.map().max()   would be easier to understand.  whenever you are using foreach to populate a local variable (e.g. max) you should think whether it can be done with assignment instead.   
const max = _().map().max().

Reply:
code is more readable and less prone to error

Reply:
Thanks!